package.json

```diff
@@ -208,28 +206,12 @@
     "plugins": [
       "@semantic-release/commit-analyzer",
       "@semantic-release/release-notes-generator",
-      [
```

Comment:
From what I read here https://github.com/cycjimmy/semantic-release-action?tab=readme-ov-file#extra_plugins

We have to keep this configuration.